test/colorado_lottery_test.rb

```diff
@@ -0,0 +1,81 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/contestant'
+require './lib/game'
+require './lib/colorado_lottery'
+
+
+class ColoradoLotteryTest < Minitest::Test
+  def setup
```

Comment:
Nice job utilizing a setup method, but it might get confusing to only use it for one type of object.